modules/addons/src/gltf/gltf-material-parser.js

```diff
@@ -137,5 +136,11 @@ export default class GLTFMaterialParser {
         ]
       });
     }
+
+    // TODO - placeholder for handling the unlit extension
+    // KHR_materials_unlit extension (postprocessed by GLTFLoader)
+    if (material.unlit) {
```

Comment:
This seems unrelated to this PR. Not a huge deal, but I'd prefer not to add logic for yet-to-be-defined APIs.